### PR TITLE
Various code cleanups

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestOrcPageSourceMemoryTracking.java [hunk 1]
```diff
@@ -749,9 +749,7 @@ public Object getWriteValue()
         {
             StringBuilder builder = new StringBuilder();
             String source = writeValue.get();
-            for (int i = 0; i < counter / step; i++) {
-                builder.append(source);
-            }
+            builder.append(String.valueOf(source).repeat(Math.max(0, counter / step)));
```
Comment (findepi) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/TestOrcPageSourceMemoryTracking.java:752]: `source` is already a String

did you mean `firstNonNull(source, "null")`?

File: presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryConnectorModule.java [hunk 2]
```diff
@@ -94,6 +95,6 @@ static String calculateBillingProjectId(Optional<String> configParentProjectId,
             return ((ServiceAccountCredentials) credentials.get()).getProjectId();
         }
         // 3. No configuration was provided, so get the default from the environment
-        return BigQueryOptions.getDefaultProjectId();
+        return ServiceOptions.getDefaultProjectId();
```
Comment (findepi) [inline presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryConnectorModule.java:98]: hm... `BigQueryOptions` seemed more appropriate (i don;t know the design behind BigQueryOptions vs generic ServiceOptions though)

File: presto-main/src/main/java/io/prestosql/operator/BigintGroupByHash.java [hunk 3]
```diff
@@ -200,7 +200,7 @@ public boolean contains(int position, Page page, int[] hashChannels)
     @Override
     public long getRawHash(int groupId)
     {
-        return BigintType.hash(valuesByGroupId.get(groupId));
+        return AbstractLongType.hash(valuesByGroupId.get(groupId));
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/operator/BigintGroupByHash.java:203]: Qualifying with `BigintType` was more appropriate.

File: presto-main/src/test/java/io/prestosql/operator/TestOperatorAssertion.java [hunk 4]
```diff
@@ -52,7 +52,7 @@ public void testToPagesWithBlockedOperator()
     {
         Operator operator = new BlockedOperator(Duration.valueOf("15 ms"));
         List<Page> pages = OperatorAssertion.toPages(operator, emptyIterator());
-        Assert.assertEquals(pages, ImmutableList.of());
+        assertEquals(pages, ImmutableList.of());
```
Comment (findepi) [inline presto-main/src/test/java/io/prestosql/operator/TestOperatorAssertion.java:55]: nit: does not belong to `Access static methods on declared class` 

(same in TestPinotClient)

File: presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryConnectorModule.java (hunk 2 above)
Comment (electrum) [reply presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryConnectorModule.java:98]: I'll drop this commit

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestOrcPageSourceMemoryTracking.java (hunk 1 above)
Comment (electrum) [reply presto-hive/src/test/java/io/prestosql/plugin/hive/TestOrcPageSourceMemoryTracking.java:752]: Thanks, this got added somehow

**Pull request merged**